### PR TITLE
Adjust formatting of RFR email contents

### PR DESCRIPTION
**Pull request by rwestberg**
Hi all,

Please review these changes that adjust the formatting of the RFR email contents, to make it more concise and also add a bit more information.

Best regards,
Robin

Progress
--------
- [x] Change must not contain extraneous whitespace
- [x] Change must be properly reviewed

Approvers
---------
 * Erik Helin ([ehelin](@edvbld) - **Reviewer**)

### CODE REVIEW

File: bots/mlbridge/src/main/java/org/openjdk/skara/bots/mlbridge/ArchiveMessages.java [hunk 1]
```diff
@@ -81,39 +76,33 @@ static String composeRebaseComment(PullRequestInstance prInstance, URI fullWebre
...
         var newCommitMessages = prInstance.formatCommitMessages(lastHead, prInstance.headHash(), ArchiveMessages::formatCommit);
         return "The pull request has been updated with additional changes.\n\n" +
                 infoSeparator + "\n\n" +
                 "Added commits:\n" +
                 newCommitMessages + "\n\n" +
-                "Pull request:\n" +
-                prInstance.pr().getWebUrl() + "\n\n" +
+                "  Stats: " + prInstance.stats(lastHead, prInstance.headHash()) + "\n" +
+                "Changes:\n\n" +
+                "  - all: " + prInstance.pr().getWebUrl() + "/files\n" +
```
Comment (kevinrushforth) [inline bots/mlbridge/src/main/java/org/openjdk/skara/bots/mlbridge/ArchiveMessages.java]: Using `/files` is GitHub-specific and won't work on GitLab which uses `/diffs`.

File: bots/mlbridge/src/main/java/org/openjdk/skara/bots/mlbridge/PullRequestInstance.java [hunk 2]
```diff
@@ -67,11 +67,40 @@ String diffUrl() {
         return pr.getWebUrl() + ".diff";
     }
 
+    String changeUrl() {
+        return pr.getWebUrl() + "/files";
```
Comment (kevinrushforth) [inline bots/mlbridge/src/main/java/org/openjdk/skara/bots/mlbridge/PullRequestInstance.java]: Needs to be `/diffs` on GitLab.

File: bots/mlbridge/src/main/java/org/openjdk/skara/bots/mlbridge/PullRequestInstance.java [hunk 3]
```diff
@@ -67,11 +67,40 @@ String diffUrl() {
...
+    }
+
+    String changeUrl(Hash base, Hash head) {
+        return pr.getWebUrl() + "/files/" + base.abbreviate() + ".." + head.abbreviate();
```
Comment (kevinrushforth) [inline bots/mlbridge/src/main/java/org/openjdk/skara/bots/mlbridge/PullRequestInstance.java]: This is done differently on GitLab (not exactly sure what the mapping would be).

**Comment by rwestberg** [comment]
Thanks Kevin, good point, this should be generalized to support GitLab as well.

**Pull request closed without merging**